e2e_tests/tests/nightly/compute_stats.py

```diff
@@ -34,6 +47,9 @@ def parse_log_for_gpu_stats(log_path: str) -> Tuple[int, str, str]:
             match_date = date_parsing_re.match(line)
             if match_date:
                 ts = iso_date_to_epoch(match_date.group(1))
+                if ts == -2:
```

Comment:
`parser.parse` will throw an exception (`dateutil.parser.ParserError`) - no need for "custom" return code